lib/mpt-trusted-light-client-types/src/consensus_state.rs

```diff
@@ -0,0 +1,43 @@
+use unionlabs::primitives::H256;
+
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct ConsensusState {
+    pub ibc_storage_root: H256,
+    pub timestamp: u64,
+}
+
+#[cfg(feature = "ethabi")]
+pub mod ethabi {
+    use alloy::sol_types::SolValue;
+    use unionlabs::impl_ethabi_via_try_from_into;
+
+    use super::*;
+
+    impl_ethabi_via_try_from_into!(ConsensusState => SolConsensusState);
+
+    alloy::sol! {
+        struct SolConsensusState {
+            bytes32 ibc_storage_root;
+            uint64 timestamp;
+        }
```

Comment:
don't we need state root and storage root here?

Reply:
state root is only used for verifying the account root. We verify the membership proofs against the storage root.

Reply:
I think we want both state root and ibc storage root so chains we only want to track the state root for state lenses we can use a dummy contract root?